Assets/Content/Items/Functional/Parts/Lights/BlinkingLight.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(MeshRenderer))]
+public class BlinkingLight : MonoBehaviour
+{
+    enum LightMode
+    {
+        StayOn,
+        Blinking,
+        Off
+    }
+    
+    private MeshRenderer lightMesh;
+    private LightMode lightMode = LightMode.Off;
+    private IEnumerator blinkCoroutine;
+    
+    [Header("Materials")]
+    [SerializeField] private Material nonEmissiveMaterial;
+    [SerializeField] private Material emissiveMaterial;
+
+    [Header("Blink Settings")]
+    [Tooltip("How often should a blink occur in seconds.")]
```

Comment:
Isn't this how many seconds between blinks?

Reply:
Very true. I updated the tooltip with clearer language.